uportal-war/src/main/java/org/jasig/portal/groups/pags/testers/EagerRegexTester.java

```diff
@@ -55,7 +57,15 @@
  * @see RegexTester
  */
 public class EagerRegexTester extends RegexTester {
-    
+
+    public EagerRegexTester(IPersonAttributesGroupTestDefinition definition) {
```

Comment:
Worth an `@since` for inline documentation of when this constructor entered the API?